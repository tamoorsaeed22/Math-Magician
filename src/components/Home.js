export default function Home() {
  return (
    <section className="message">
      <h2>Welcome to our page!</h2>
      <p>
        It is my pleasure to welcome you to the world of mathematics where numbers come to live.
        We offer a variety of tools and collection of random computer quotes to entertain you.
        Our easy-to-use calculators are designed to help you understand and solve math concepts.
        So come on in and explore all that Math Magicians has to offer.
      </p>
    </section>
  );
}
