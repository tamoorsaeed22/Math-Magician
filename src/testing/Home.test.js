import renderer from 'react-test-renderer';
import Home from '../components/Home';

it('renders homepage correctly', () => {
  const tree = renderer.create(
    <Home />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
