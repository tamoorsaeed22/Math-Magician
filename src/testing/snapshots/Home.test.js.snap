// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage correctly 1`] = `
<section
  className="message"
>
  <h2>
    Welcome to our page!
  </h2>
  <p>
    It is my pleasure to welcome you to the world of mathematics where numbers come to live. We offer a variety of tools and collection of random computer quotes to entertain you. Our easy-to-use calculators are designed to help you understand and solve math concepts. So come on in and explore all that Math Magicians has to offer.
  </p>
</section>
`;